src/qt/trafficgraphwidget.h

```diff
@@ -37,7 +35,7 @@ public Q_SLOTS:
     void clear();
 
 private:
-    typedef boost::function<float(const TrafficSample&)> SampleChooser;
+    typedef std::function<float(const TrafficSample&)> SampleChooser;
```

Comment:
nit: that's dash code, let's replace `typedef` to `using` if touching this line.
Semantically it is the same, but easier to read.

```suggestion
    using SampleChooser = std::function<float(const TrafficSample&)>;
```

Reply:
Changes implemented, resolved in latest push!